src/core/mac/mac.cpp

```diff
@@ -748,7 +768,9 @@ void Mac::HandleBeginTransmit(void)
 
 extern "C" void otPlatRadioTransmitDone(otInstance *aInstance, bool aRxPending, ThreadError aError)
 {
+    otLogFuncEntryMsg("%!otError!, aRxPending=%u", aError, aRxPending ? 1 : 0);
```

Comment:
Actually, since `otLogFuncEntryMsg` is never defined on anything but Windows, do you mind just leaving this for now. If we ever implement this on a non-Windows platform, we can fix it then?